pghrep/templates/raw.tpl

```diff
@@ -0,0 +1,12 @@
+Current values
+===
+
+Available only raw data.
+
+{{ range $host, $data := .rawData }}
+    Host: {{ (index $data "host") }}
+    ``` {{ (index $data "data") }} ```
```

Comment:
This template will look like this example:

    Host: dev
    ``` bla bla bla ```

May be fix to this variant? : 

Host: `dev`

```
Some collected
data with help of checker
```

Host: `next.host`

```
Blablabla
```

Reply:
You can edit my comment a use github's preview and play with styles.

Reply:
But we decided to fix it in #121